nb/regression/shape/shape.nb

```diff
@@ -192,3 +192,11 @@ Arc(2)
   .view();
 
 Triangle(14).svg('triangle');
+
+Box(10)
+  .ez(50, -50)
+  .rx(0, 1 / 4)
+  .fuse().view(0)
+  .smooth({ method: 'IsotropicRemeshing', iterations: 20, relaxationSteps: 10, targetEdgeLength: 0.5, }, Box(20, 20, 20))
```

Comment:
So the Box(20, 20, 20) argument to .smooth() is doing the selection? I like that a lot.

I was picturing something that worked like a.select(Box(20,20,20)).smooth() when you described it (I got your message but didn't have service to reply), but that has all sorts of tricky issues about how it would actually work.

Reply:
Yes, some kind of sticky section could be done, but I think as long as we can make the following work, we should be fine.

a.and(Box(20, 20, 20).opaque(0.3).as('selection')).smooth(get('selection')).view()

Reply:
Yeah, that looks nice and should work now, right?